test/models/adoption_application_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AdoptionApplicationTest < ActiveSupport::TestCase
```

Comment:
Since you haven't written any tests here, how would you feel about removing this file?